### PR TITLE
Add SauceLabs promos. Bug 1330094.

### PR DESCRIPTION
**Pull request by jpetto**
We are still waiting on copy approval, but code is good for review.

Note - depends on [kumascript PR 127](https://github.com/mozilla/kumascript/pull/127).

@schalkneethling - Have time to check this out Monday or Tuesday next week?

### CODE REVIEW

File: kuma/static/styles/components/home/column-callout.scss [hunk 1]
```diff
@@ -135,3 +105,127 @@ Colours and icons
...
+    .column-callout {
+        margin-bottom: $grid-spacing;
+    }
+
+    .column-callout a:after {
+        top: $callout-icon-spacing;
+        @include bidi-style(right, $callout-icon-spacing + $grid-spacing, left, auto);
+    }
+
+    .callout-saucelabs a {
```
Comment (schalkneethling) [inline kuma/static/styles/components/home/column-callout.scss]: Nit: On line 172 above, this is written as:

```
.callout-saucelabs {
  a {
```

The resulting CSS will be a little different i.e.

```
.callout-saucelabs a { ... };
.callout-saucelabs span { ... };
```
 vs.

```
.callout-saucelabs a { ... };
.callout-saucelabs a span { ... };
```
 Same end result, just nit picking ;)

File: kuma/landing/jinja2/landing/homepage.html [hunk 2]
```diff
@@ -122,8 +122,14 @@ <h2 class="offscreen">{{ _('Open Web and Mobile') }}</h2>
     <div class="column-callout callout-learn">
       <a href="{{ wiki_url('Learn') }}"><span>{{ _('Learn Web Development') }}</span></a>
     </div>
-    <div class="column-callout callout-deved">
-      <a href="https://www.mozilla.org/firefox/developer/?utm_source=developer.mozilla.org&utm_medium=referral&utm_campaign=mdn-front-pg-promo"><span>{{ _('Get The Browser For Developers Like You') }}</span></a>
+    <div class="column-callout callout-saucelabs">
+      <a href="https://saucelabs.com/cross-browser-testing-tutorial">
+        <span>
```
Comment (schalkneethling) [inline kuma/landing/jinja2/landing/homepage.html]: Do you need to introduce this `span`? Can the `width` style rules not just be applied to the `a`?

File: kuma/static/styles/components/home/column-callout.scss (hunk 1 above)
Comment (jpetto) [inline kuma/static/styles/components/home/column-callout.scss]: Will update 😁

File: kuma/landing/jinja2/landing/homepage.html (hunk 2 above)
Comment (jpetto) [inline kuma/landing/jinja2/landing/homepage.html]: This `<span>` keeps the text from overlapping the background image applied to the `<a>`.

**Comment by jpetto** [comment]
> I am guessing we need to get mozilla/kumascript#127 merged before I can actually test this locally correct?

Nope! Since kumascript is a separate repo, you can check out this branch *and* that branch at the same time. Or, you can just check out this branch and create (copy/paste) the necessary file from the kumascript PR.

**Comment by jpetto** [comment]
@schalkneethling - Updated copy on the home page promo, which required a little CSS/markup work. Kept as a separate commit for easy reviewing. Will/should squash before merging.

Thanks!

**Comment by schalkneethling** [comment]
A couple of issue at different break points. So, at what is defined as `breakDesktop` in bedrock the graphic is cut of on the right hand side:

breakdesktop

At `breakTablet aka 760x1280` it is fine however as the promos start to stack vertically. Then at `breakMobileLandscape` and smaller the graphic is again cur of a little on the left hand(I guess it would be right hand side as well for rtl) side:

breakmobilelandscape

Not sure if this is a big problem but, wanted to make note of it. Other than that, everything looks good. The only thing I am unclear about is, where would I copy and paste the code from kumascript into the local version of kuma?

There is a `kumascript` folder at the root of kuma but, it does not contain the `macros` folder that lives here: https://github.com/mozilla/kumascript/tree/master/macros

**Comment by jpetto** [comment]
Yeah, the graphic is moved off to the right a bit at a few breakpoints to ensure the text does not overlap (and thereby become difficult to read). It isn't ideal, but was the best option given the image and responsive constraints.

**Comment by jpetto** [comment]
@schalkneethling Added one more commit that changes the order of the home page promos based on a [request in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1330094#c35).

If all looks good to you, feel free to "squash and merge" with the green button (and on the kumascript PR).

Thanks!

**Comment by schalkneethling** [comment]
r+

**Comment by schalkneethling** [comment]
@jpetto Squashed and merged. I do not have the required rights to merge on the kumascript repo so, merge at will.

**Pull request merged**